### DISCUSSION TITLE
Using Beta v2.1.0b13, when I run through Home Assistant, it takes me to the settings page instead of home page

### DISCUSSION THREAD

**Discussion by harryfine** [music-assistant/support#2613] [Q&A] [answered]
Using HA latest version, installed as HAOS on an Intel NUC.

Using MA Beta addon v2.1.0b15, when I run through Home Assistant, it takes me to the settings page instead of home page.  I have to hit Home to take me back to the Home screen whenever I run the software.  Is this a known bug in v.2.1.0b15?

**Comment by OzGav** [comment, marked as answer]
https://github.com/music-assistant/hass-music-assistant/issues/2491

**Reply by harryfine** [reply]
The thread showed had someone mention that it was fixed in b15 but it is not.

**Reply by OzGav** [reply]
Which is what the last two comments in that thread say
